mycroft/audio/service.py

```diff
@@ -238,3 +248,4 @@ def init_messagebus(self):
         self.bus.on('mycroft.audio.speech.stop', self.handle_stop)
         self.bus.on('mycroft.audio.queue', self.handle_queue_audio)
         self.bus.on('speak', self.handle_speak)
+        self.bus.on('neon.get_languages_tts', self.handle_get_languages_tts)
```

Comment:
can we change this message? let's also try to standardize the message a little bit `ovos.languages.{service}.get`?